solr/conf/solrconfig.xml

```diff
@@ -170,19 +170,18 @@
          additional_title_display_ssm,
          format,
          publication_display_ssm,
-         pub_date_ssim,
+         pub_date_ssm,
          overall_imprint_display_ssm,
          copyright_display_ssm,
          edition_display_ssm,
          phys_desc_ssm,
-         language_facet_ssim,
+         language_ssim,
```

Comment:
@cdmo should this be `language_facet`? 🤔

Reply:
the facet is now docvalues. For the sake of just testing docvalues, I split language into two fields, one for faceting and one for index and display. Same for pub date